### PR TITLE
Add ErrorBoundary component in catalog and instances ui

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Add `ErrorBoundary` component in catalog and instances ui for error handling: 
  - fix bug in catalog `CreateInstanceModal` with `instanceCreateParameterSchema` prop,
  - fix bug in instances `CreateCredentialsModal` with `bindingCreateParameterSchema` prop,

**Related issue(s)**
See also https://github.com/kyma-project/kyma/issues/1648

### CODE REVIEW

File: catalog/src/components/ServiceClassDetails/CreateInstanceModal/SchemaData.component.js [hunk 1]
```diff
@@ -37,16 +42,29 @@ class SchemaData extends React.Component {
     } = this.props;
     const { instanceCreateParameters } = this.state;
 
+    const schema =
+      typeof instanceCreateParameterSchema === 'string'
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/CreateInstanceModal/SchemaData.component.js]: Is there really a case, when `instanceCreateParameterSchema` is string?

File: catalog/src/components/ServiceClassDetails/CreateInstanceModal/SchemaData.component.js [hunk 2]
```diff
@@ -37,16 +42,29 @@ class SchemaData extends React.Component {
...
-      <JsonSchemaForm
-        schema={instanceCreateParameterSchema}
-        onChange={this.onChangeSchemaForm}
-        liveValidate={true}
-        onSubmit={onSubmitSchemaForm}
-        formData={instanceCreateParameters}
+      <ErrorBoundary
+        content={
+          <div>
+            <Icon icon={'\uE1EC'} /> Incorrect schema
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/CreateInstanceModal/SchemaData.component.js]:
```
            <Icon icon={'\uE1EC'} /> Incorrect Instance Create Parameter schema in plan X
```

File: instances/src/components/ErrorBoundary/ErrorBoundary.component.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
```
Comment (pkosiec) [inline instances/src/components/ErrorBoundary/ErrorBoundary.component.js]: Please move it to `react-components`

File: catalog/src/components/ServiceClassDetails/CreateInstanceModal/SchemaData.component.js [hunk 4]
```diff
@@ -37,16 +42,29 @@ class SchemaData extends React.Component {
     } = this.props;
     const { instanceCreateParameters } = this.state;
 
+    const schema =
+      typeof instanceCreateParameterSchema === 'string'
+        ? JSON.parse(JSON.stringify(instanceCreateParameterSchema))
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/CreateInstanceModal/SchemaData.component.js]: Even if the case occurs, why do you stringify a string?

File: instances/src/components/ServiceInstanceDetails/ServiceInstanceBindings/CreateCredentialsModal/SchemaData.component.js [hunk 5]
```diff
@@ -37,16 +42,29 @@ class SchemaData extends React.Component {
     } = this.props;
     const { bindingCreateParameters } = this.state;
 
+    const schema =
+      typeof bindingCreateParameterSchema === 'string'
+        ? JSON.parse(JSON.stringify(bindingCreateParameterSchema))
```
Comment (pkosiec) [inline instances/src/components/ServiceInstanceDetails/ServiceInstanceBindings/CreateCredentialsModal/SchemaData.component.js]: Why do you stringify a string?

File: instances/src/components/ServiceInstanceDetails/ServiceInstanceBindings/CreateCredentialsModal/SchemaData.component.js [hunk 6]
```diff
@@ -37,16 +42,29 @@ class SchemaData extends React.Component {
     } = this.props;
     const { bindingCreateParameters } = this.state;
 
+    const schema =
+      typeof bindingCreateParameterSchema === 'string'
```
Comment (pkosiec) [inline instances/src/components/ServiceInstanceDetails/ServiceInstanceBindings/CreateCredentialsModal/SchemaData.component.js]: Is there really a possibility to get a string?

File: instances/src/components/ServiceInstanceDetails/ServiceInstanceBindings/CreateCredentialsModal/SchemaData.component.js [hunk 7]
```diff
@@ -37,16 +42,29 @@ class SchemaData extends React.Component {
...
-      <JsonSchemaForm
-        schema={bindingCreateParameterSchema}
-        onChange={this.onChangeSchemaForm}
-        liveValidate={true}
-        onSubmit={onSubmitSchemaForm}
-        formData={bindingCreateParameters}
+      <ErrorBoundary
+        content={
+          <div>
+            <Icon icon={'\uE1EC'} /> Incorrect schema
```
Comment (pkosiec) [inline instances/src/components/ServiceInstanceDetails/ServiceInstanceBindings/CreateCredentialsModal/SchemaData.component.js]:
```
<Icon icon={'\uE1EC'} /> Incorrect Binding Create Parameter schema in plan X
```

Comment (magicmatatjahu) [reply instances/src/components/ServiceInstanceDetails/ServiceInstanceBindings/CreateCredentialsModal/SchemaData.component.js]: In bindings?

Comment (pkosiec) [reply instances/src/components/ServiceInstanceDetails/ServiceInstanceBindings/CreateCredentialsModal/SchemaData.component.js]: Yup

File: README.md [hunk 8]
```diff
@@ -23,16 +23,16 @@ Console also includes the libraries with components for the React and Angular fr
...
-4. Run the following command to run the Console with [`core`](./core) and all the views locally:
-```
-npm run start
-```
-5. To access the local instance of the Console at http://console-dev.kyma.local:4200, use credentials from [this](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/031-gs-local-installation.md#access-the-kyma-console) document.
+    ```
+    npm install
+    npm run bootstrap
+    ```
+    **NOTE:** `npm run bootstrap` command updates your `/etc/hosts` with `127.0.0.1 console-dev.kyma.local` host.
```
Comment (kazydek) [inline README.md]:
```suggestion
    **NOTE:** The `npm run bootstrap` command updates your `/etc/hosts` with the `127.0.0.1 console-dev.kyma.local` host.
```

**Pull request merged**